WebHostLib/__init__.py

```diff
@@ -70,6 +70,13 @@ def to_url(self, value):
 app.jinja_env.filters['suuid'] = lambda value: base64.urlsafe_b64encode(value.bytes).rstrip(b'=').decode('ascii')
 
 
+def get_world_theme(game_name: str):
+    for game, world in AutoWorldRegister.world_types.items():
```

Comment:
You can just lookup instead of iterating; `AutoWorldRegister.world_types[game_name].web.theme`

Reply:
This one has been fixed and pushed.